docs/trace/building-your-own-exporter/MyExporter.cs

```diff
@@ -19,13 +19,16 @@
 using System.Diagnostics;
 using System.Threading;
 using System.Threading.Tasks;
+using OpenTelemetry;
 using OpenTelemetry.Trace;
 
 internal class MyExporter : ActivityExporter
 {
     public override Task<ExportResult> ExportAsync(
         IEnumerable<Activity> batch, CancellationToken cancellationToken)
     {
+        using var scope = Sdk.SuppressInstrumentation.Begin();

```

Comment:
should we add a comment here what this line does and why is it important?

Reply:
I was struggling and decided not here, instead, will put it in the doc (once I start to work on doc again).